.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+<!-- Short description of the PR. What does it do? -->
+This PR implements a new data enhancement to...
+
+## Categories
+<!-- Select the categories that your PR better fits on -->
+
+- [ ] Bugfix
+- [X] Enhancement
+- [ ] Backporting
+- [ ] Others (CI, Infrastructure, Documentation)
+
+## Sample archive
```

Comment:
This can be checkbox as well right.

Reply:
I mean I would really put it as a question.

Reply:
Hummm... If it doesn't have any sample archive, just put nothing there, if it has put the sample data path. Don't think that add a checkbox or just a question would be helpful. Otherwise, we should do it for all the other lists.